### DISCUSSION TITLE
How to close RemixAI, the new Web3 AI Assistant

### DISCUSSION THREAD

**Discussion by AnaAse** [Cyfrin/foundry-full-course-cu#2876] [Q&A]
Im on a Macbook and need help to close the new RemixAI assistant which auto-opens when you click on the REMIX link and I cannot find any X buttom or similiar to minimize it. Thank you!
<img width="1237" alt="Screenshot 2024-11-17 at 16 07 11" src="https://github.com/user-attachments/assets/4cbca15e-431b-414b-93ca-5d0604c3e35a">

**Comment by EngrPips** [comment]
Hello @AnaAse, You can click on any file you want to work on, and it will open the file. You can also see the arrow to click on to close the RemixAI in the image below

![Screenshot 2024-11-17 at 21 13 11](https://github.com/user-attachments/assets/ea9e98a8-e988-491f-b4db-b1626e6a1887)

**Reply by LordAdhaar** [reply]
Indeed

**Comment by LordAdhaar** [comment]
Try zooming out if you are unable to see the dropdown
Cmd/Ctrl plus - key
Or
Click on the three dots in the top right corner and zoom out

**Comment by AnaAse** [comment]
Thank you all!

Since zooming did not work, I tried to figure out where the down arrows shall be located (but not visible) and as the pointer turned to a hand... Clicked and it worked.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Comment by Jainam012** [comment]
I am facing the same issue i am on windows. It's a small problem but really frustrating. I am not able to solve it can anyone help me?

**Reply by EngrPips** [reply]
What issue are you facing?

**Reply by Jainam012** [reply]
I fixed it. I was not able to find the down arrow the close the ai chat nor it was becoming small by dragging. I saved my code and cleared the cookies then it was all okay. 

Thanks for your concern.

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.
